packages/opensearch/src/fixIds.ts

```diff
@@ -45,8 +45,8 @@ export const fixIds = <T extends WithId, K = T>(
       terms: q.terms ? fixId(q.terms) : undefined,
       wildcard: q.wildcard ? fixId(q.wildcard) : undefined,
     },
-    _source: searchQuery.body._source,
-  } as NativeOpenSearchQueryBody<NativeOpenSearchType<T>, K>
+    ...bodyRest,
+  } as unknown as  NativeOpenSearchQueryBody<NativeOpenSearchType<T>, K>
```

Comment:
The `as unknown` will make the object loose type safety here. No warnings would be thrown for:

```typescript
const body = {
  query: {
    bool: q.bool ? fixBool(q.bool) : undefined,
    // Rest of the fields...
    test: 'this-is-not-a-real-field',
  },
  ...bodyRest,
  notReal: 'neither-is-this',
} as unknown as NativeOpenSearchQueryBody<NativeOpenSearchType<T>, K>
```

Reply:
I know... I am looking at it right now to solve it in a better way. I just wanted the functionality into prod so that @alexanderczigler can move on while I fix it

Reply:
I couldn't find a good solution right away either. Should `OpenSearchType` and `NativeOpenSearchType` line up? `OpenSearchType` has `query.from` and `query.size` that the other doesn't have, and `NativeOpenSearchType` has `query.fields` that the other one doesn't have.